search-api/.env.dev

```diff
@@ -0,0 +1,7 @@
+
+## DB SETTINGS 
+export POSTGRES_DB_HOST="localhost"
```

Comment:
In this PR: https://github.com/cuebook/CueObserve/pull/159, I have modified the way we are using environment variables. So, here as well, I think we should use the same approach. We have a single `.env` file in the root of our repository, this file is recognized by docker-compose automatically. We have to add search-api Environment Variables in global .env as well.

Since we also have other database env variables in .env file, we can name the variables something like this: `SEARCH_DB_HOST`.